### PR TITLE
Add site launch runbook

### PR DESCRIPTION
**Pull request by brown-a2**

### CODE REVIEW

File: runbooks/site-launch.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Required
+
+* WASM
+* AWS access
+* You'll also need to be connected to the VPN.
+
+## Phases
+
+This assumes all contnet is ready on the production environment.
```
Comment (RobLoweMOJ) [inline runbooks/site-launch.md]: *content

File: runbooks/site-launch.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+### Phase II - Steps at time of launch
+1. Deactivate `Force Login` plugin
+2. Login to prod, change WP `Siteurl` and `Home` to the domain (for example add, https://ccrc.gov.uk).
+3. Check site links are pointing to domain not jotwpublic.prod. If the links are wrong, you may need to run a wp search and replace command on the database. To do so, login to prod via WASM and run, for example `wp search-replace "jotwpublic.prod.wp.dsd.io/ccrc" "ccrc.gov.uk" --url=ccrc.gov.uk --dry-run` . Make sure to take the `--dry-run` when you want the changes to take effect.
+4. Confirm with domains team repoint is done.
+
+
+## Troubleshooting
+
+After launching the site, some changes may not seem to be taking effect. Domain changes can take some time to propagate so it maybe be that not enough time has passed. Further, there may be caching issues happened. Try clearing the cache on the problem page.
```
Comment (RobLoweMOJ) [inline runbooks/site-launch.md]: Maybe add here how to clear server cache?

**Review by RobLoweMOJ — APPROVED** [review]

File: runbooks/site-launch.md (hunk 1 above)
Comment (brown-a2) [reply runbooks/site-launch.md]: Done 👍

File: runbooks/site-launch.md (hunk 2 above)
Comment (brown-a2) [reply runbooks/site-launch.md]: Added further info.

**Pull request merged**